src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -25,9 +25,19 @@ export const CheckImg = styled.img`
   }
 
   @media (min-width: 1025px) {
-    margin-left: 360px;
-    margin-top: 185px;
-    width: 150px;
+    margin-left: auto;
+    position: relative;
+    left: 266px;
+    top: 232px;
```

Comment:
I would suggest not to use big static values to position HTML elements, this is not predictable for other developers that want to collaborate with you. And this way of doing things is not adaptable for other screen sizes than yours. But how desktop is not the objective of this pull request I think this is not needed to resolve for now. Maybe in another PR.